Report/sections/1_analysis/Behavior.tex

```diff
@@ -0,0 +1,19 @@
+\chapter{Behavior}\label{chap:behavior}
+
+As concluded in the last chapter, learning how to program for the first time is hard.
+Therefore, a natural next question would be to ask how one could make it easier.
+That is the topic this chapter will examine. \\
+
+Earlier, in \chapref{chap:analysis}, we presented \textit{deliberate practice} as an optimal way to learn.
+However, merely knowing how to learn is not the same as learning. \\
+
+For any behavior to occur, three elements must be present. Namely, motivation, ability to perform a given behavior, and a prompt \cite{bjfoggBehaviorModelPersuasive2009}. If any of those are not present, the behavior will not occur. \\
```

Comment:
This feels a little sudden and out of context. Maybe make a short introduction to behavioral psychology before presenting this as a cold hard fact?